modules/imgcodecs/test/test_tiff.cpp

```diff
@@ -46,6 +46,201 @@ TEST(Imgcodecs_Tiff, decode_tile16384x16384)
     EXPECT_EQ(0, remove(file4.c_str()));
 }
 
+#ifdef __ANDROID__
+// Test disabled as it uses a lot of memory.
+// It is killed with SIGKILL by out of memory killer.
+TEST(Imgcodecs_Tiff, DISABLED_decode_issue22388_16UC1)
+#else
+TEST(Imgcodecs_Tiff, decode_issue22388_16UC1)
+#endif
+{
+    // See https://github.com/opencv/opencv/issues/22388
+    string file3 = cv::tempfile(".tiff");
```

Comment:
why is "3"?

Reply:
Thank you for your review ! Based on this code, I implemented this test.

https://github.com/opencv/opencv/blob/3.4/modules/imgcodecs/test/test_tiff.cpp#L25-L26
> string file3 = cv::tempfile(".tiff");
> string file4 = cv::tempfile(".tiff");

I'll fix it, thank you !!

Reply:
I think it has been fixed.

https://github.com/opencv/opencv/pull/22404/files#diff-a28d9eb23f9748f98627df2c05ca83c536df9513008fcbfb30bfe0821dbeec47R121-R123

```c++
    // Detect data file
    const string req_filename = cv::format("readwrite/huge-tiff/%s_%llu.tif", mat_type_string.c_str(), buffer_size);
    const string filename = findDataFile( req_filename );
```